packages/docs/src/App.vue

```diff
@@ -34,7 +37,13 @@ export default defineComponent({
         duration: 3000,
       })
     }
+
     useColors().setColors(COLOR_THEMES[ThemeName.DEFAULT])
+
+    onMounted(() => {
+      const currentLocale = getCurrentInstance()?.ctx?.$i18n?.locale
```

Comment:
Because of `<html lang="en">` in [packages/docs/public/index.html](https://github.com/epicmaxco/vuestic-ui/pull/2129/files/39ceff292d8150eab1abc83dc21aa3fa17fd957a#diff-ba992d56b3e6501bec5f60167498b3160038e4c703e69d616df79d7863d8f3a2) this looks redundant.